pkg/kubernetes/client/get.go

```diff
@@ -0,0 +1,78 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/grafana/tanka/pkg/kubernetes/manifest"
+)
+
+// Get retrieves a single Kubernetes object from the cluster
+func (k Kubectl) Get(namespace, kind, name string) (manifest.Manifest, error) {
+	m, err := k.get(namespace, []string{kind, name})
```

Comment:
Seems like you can just replace all of this with `return k.get(namespace, []string{kind, name})`